Requirements/architecture-requirements.md

```diff
@@ -0,0 +1,89 @@
+# Demo Architecture
+
+## Summary
+
+The OpenTelemetry Community Demo application is intended to be a ‘showcase’ for
+OpenTelemetry API, SDK, and tools in a “production-ish” cloud native
```

Comment:
Maybe instead of this vague word in quotes we add a footnote to the word production?

Reply:
Switched to production-lite phrasing